Source/Extensions/Blazorise.Cropper/wwwroot/blazorise.cropper.js

```diff
@@ -298,6 +294,22 @@ export function resetSelection(element, elementId) {
     }
 }
 
+function cropperImageReady(cropperImage, cropperCanvas, instance) {
+    cropperImage.$ready((image) => {
+        if (instance.wasDisabled) {
```

Comment:
The `wasDisabled` seems strange in naming. Was, and Disabled means something in the past. I would name it either simply `disabled` or something more descriptive of what it does. Eg. `disabledOnLoad`, or `disabledOnError`.